### PR TITLE
Changes for Junebug

### PR DESCRIPTION
**Pull request by FredJEhlers**

### CODE REVIEW

File: oneplusmvp/settings.py [hunk 1]
```diff
@@ -182,10 +183,20 @@ def abspath(*args):
...
+
+# Junebug config
+JUNEBUG_BASE_URL = os.environ.get("JUNEBUG_BASE_URL", "https://junebug.qa.dig-it.me/jb/channels/")
+JUNEBUG_CHANNEL_ID = os.environ.get("JUNEBUG_CHANNEL_ID", "ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1/")
+JUNEBUG_ACCOUNT_NUMBER = os.environ.get("JUNEBUG_ACCOUNT_NUMBER", "")
+JUNEBUG_FAKE_SMS = False
+JUNEBUG_FROM = "*11910"
+JUNEBUG_FAKE_TO = "0660000000"
+JUNEBUG_USERNAME = "junebug"
+JUNEBUG_PASSWORD = "DbAOTBsm4zXbU10x"
```
Comment (miltontony) [inline oneplusmvp/settings.py]: Please leave all these fields blank. Secondly, they should be read from ENV

File: oneplusmvp/settings.py [hunk 2]
```diff
@@ -182,10 +183,20 @@ def abspath(*args):
...
-VUMI_GO_ACCOUNT_KEY = "b365f245538841a08586a29b5b568c6c"
-VUMI_GO_ACCOUNT_TOKEN = "vaiToa6c"
+# VUMI_GO_BASE_URL = "http://go.vumi.org/api/v1/go/http_api_nostream"
+# VUMI_GO_CONVERSATION_KEY = "d5d981d5b86c4fee99f6ee078d0a0abd"
+# VUMI_GO_ACCOUNT_KEY = "b365f245538841a08586a29b5b568c6c"
+# VUMI_GO_ACCOUNT_TOKEN = "vaiToa6c"
+
+# Junebug config
+JUNEBUG_BASE_URL = os.environ.get("JUNEBUG_BASE_URL", "https://junebug.qa.dig-it.me/jb/channels/")
+JUNEBUG_CHANNEL_ID = os.environ.get("JUNEBUG_CHANNEL_ID", "ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1/")
```
Comment (miltontony) [inline oneplusmvp/settings.py]: These should preferably be blank or dummy data

File: oneplusmvp/settings.py [hunk 3]
```diff
@@ -182,10 +183,20 @@ def abspath(*args):
...
 
 # Base url for vumi requests
-VUMI_GO_BASE_URL = "http://go.vumi.org/api/v1/go/http_api_nostream"
-VUMI_GO_CONVERSATION_KEY = "d5d981d5b86c4fee99f6ee078d0a0abd"
-VUMI_GO_ACCOUNT_KEY = "b365f245538841a08586a29b5b568c6c"
-VUMI_GO_ACCOUNT_TOKEN = "vaiToa6c"
+# VUMI_GO_BASE_URL = "http://go.vumi.org/api/v1/go/http_api_nostream"
+# VUMI_GO_CONVERSATION_KEY = "d5d981d5b86c4fee99f6ee078d0a0abd"
+# VUMI_GO_ACCOUNT_KEY = "b365f245538841a08586a29b5b568c6c"
+# VUMI_GO_ACCOUNT_TOKEN = "vaiToa6c"
```
Comment (miltontony) [inline oneplusmvp/settings.py]: I don't know why these details were ever committed into a public repo. Instead of commenting out, please delete

**Review by miltontony — CHANGES_REQUESTED** [review]
Similar review to MobileU. Please remove the hardcoded values

**Review by miltontony — APPROVED** [review]
I think my comments have been resolved. @rudigiesler happy for you to give this a once over.
Travis is still failing though.

File: oneplus/tests/test_general_tests.py [hunk 4]
```diff
@@ -324,9 +325,9 @@ def test_terms(self):
         resp = self.client.post(reverse('misc.terms'), follow=True)
         self.assertEquals(resp.status_code, 200)
 
-    def assert_in_metric_logs(self, metric, aggr, value):
+    def assert_in_metric_logs(self, metric, aggr, value, temp_handler):
```
Comment (rudigiesler) [inline oneplus/tests/test_general_tests.py]: I seems like we've removed all calls to this function, so we can probably also just remove this function.

File: oneplus/utils.py [hunk 5]
```diff
@@ -20,9 +20,9 @@ def update_metric(name, value, metric_type):
 def async_sender(name, value, metric_type):
     try:
         sender = HttpApiSender(
-            account_key=settings.VUMI_GO_ACCOUNT_KEY,
-            conversation_key=settings.VUMI_GO_CONVERSATION_KEY,
-            conversation_token=settings.VUMI_GO_ACCOUNT_TOKEN
+            username=settings.JUNEBUG_USERNAME,
+            password=settings.JUNEBUG_PASSWORD,
+            api_url=settings.JUNEBUG_BASE_URL+settings.JUNEBUG_CHANNEL_ID+settings.JUNEBUG_ACCOUNT_NUMBER
```
Comment (rudigiesler) [inline oneplus/utils.py]: The `HttpApiSender` doesn't take in the parameters `username`, `password`, and `api_url`: https://go-http-api.readthedocs.io/en/latest/messaging-api.html#go_http.send.HttpApiSender .

Also, Junebug has a different API to Vumi Go, so it won't understand requests from the Vumi Go client. It also doesn't have a metric store, nor does it have a way to receive metrics, so we shouldn't be sending any metrics to it.

File: ve26/djcelery/static/djcelery/style.css [hunk 6]
```diff
@@ -0,0 +1,4 @@
+.form-row.field-traceback p {
+    font-family: monospace;
+    white-space: pre;
+}
```
Comment (rudigiesler) [inline ve26/djcelery/static/djcelery/style.css]: These seem to be files included from a virtual environment. We probably don't want to commit any files from the virtual environment into the repository

**Review by rudigiesler — COMMENTED** [review]
A few comments, I think the big one is around metrics. I think the fact that the metric calls are being mocked means that the tests are passing, but when we actually try to run this, it will fail.

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**